client/my-sites/domains/domain-management/components/domain-notice/index.jsx

```diff
@@ -14,7 +14,7 @@ import './style.scss';
 export default class DomainNotice extends React.Component {
 	static propTypes = {
 		text: PropTypes.string,
```

Comment:
I know this was already like this, but just noticed it while testing. Text that includes react elements is passed in as an array, so I guess we should allow that here too.
```suggestion
		text: PropTypes.oneOfType( [ PropTypes.string, PropTypes.array ] ),
```

Reply:
Good catch, I saw the warnings in the console but forgot to fix this 🙇